src/index.css

```diff
@@ -14,13 +14,20 @@ code {
 }
 
 .bg{
+  height: 88vh;
```

Comment:
É melhor deixar classes com responsabilidade única. I.e., o `.bg` somente setar um background, sem se preocupar com tamanho.
Se o tamanho for necessário, usar classes do Tachyons, como `.vh-100` e `vw-100`.